### ISSUE TITLE
[Capricorn] Multithread solution with factory and plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kimi-Arthur/pimix-software-suite#2]
```
Split plugin into factory and plugin and make it to work with Console
```

Original issue reported on code.google.com by `kimi.rib...@gmail.com` on 15 Aug 2013 at 1:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `kimi.rib...@gmail.com` on 15 Aug 2013 at 1:51
- Added labels: **Capricorn**

**Comment by GoogleCodeExporter** [comment]
```
Solved in changeset 98020e906880.
```

Original comment by `kimi.rib...@gmail.com` on 21 Aug 2013 at 5:08
- Changed state: **Verified**
